packages/bilby_cython/meta.yaml

```
@@ -0,0 +1,21 @@
+package:
+  name: bilby_cython
```

```suggestion
  name: bilby.cython
```

(Nit): the package's name is `bilby.cython` on PyPI, could we switch to that? It will be normalised at the time of building and change to `bilby_cython` anyway.

---

Changing the directory name from bilby_cython to bilby.cython as well should work I think